### ISSUE TITLE
Tooltips on multiBarChart do not hide after focus is lost

### ISSUE THREAD

**Issue by david-bergman** [krispo/angular-nvd3#280]
![image](https://cloud.githubusercontent.com/assets/13984154/11210607/ff55dbce-8d31-11e5-99bc-395d0df8c85b.png)

I have a dashboard with multiple Bar charts on, it seems the tooltips for quite a large percentage of the time are not removed once the mouse loses focus.

**Comment by krispo** [comment]
try to play with `hideDelay` option:
```js
chart: {
    tooltip: {
        hideDelay: 0 // default 400
    }
}
```

**Comment by david-bergman** [comment]
Thanks so much for the quick response!, It is much better now when moving between graphs, but if I stay on a single chart series for long enough and then move, the tooltip seems to remain for good until I refresh the browser, not sure if there is something that can be done in this case...

![image](https://cloud.githubusercontent.com/assets/13984154/11239450/c42d5892-8df4-11e5-9f97-d0ba6dcddcfd.png)

**Comment by krispo** [comment]
which version? tooltip issue is fixed in the latest version (1.0.3). check it please.

**Comment by david-bergman** [comment]
Upgraded from the previous version, all sorted, thanks once again!
